src/main/java/gregtech/api/fluids/MaterialFluid.java

```diff
@@ -47,7 +47,15 @@ public String getUnlocalizedName() {
     @Override
     @SideOnly(Side.CLIENT)
     public String getLocalizedName(FluidStack stack) {
-        String localizedName = I18n.format(getUnlocalizedName());
+        String localizedName;
+        String customTranslationKey = "fluid." + material.getUnlocalizedName();
```

Comment:
Shouldn't this also have a mod id check, since all our material keys are preferenced by modid now?

Or would the custom key end up being `fluid.gregtech.rubber` or something like that?

Reply:
`Material#getUnlocalizedName` returns `"modid.material.material_name"`, so this string will end up as `"fluid.modid.material.material_name"`. 

For example: `"fluid.gregtech.material.copper"`.